### PR TITLE
scan contents of a directory

### PR DESCRIPTION
**Pull request by d-grossman**
perform clamav, and a few hashes on files, return a dictionary of discoveries.

### CODE REVIEW

File: check_files/run.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+def av_results(p):
+    p = subprocess.Popen(['clamscan','--no-summary',p],stdout=subprocess.PIPE,stderr=subprocess.PIPE)
+    a = p.stdout.read()
+    return a
+    
+
+def main():
+    matches = []
+    ret_val = dict()
+    for root, dirnames, filenames in os.walk('/data'):
```
Comment (cglewis) [inline check_files/run.py]: should this be just matching on the one file that was given at `argv[1]`?

File: check_files/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+watchdog
```
Comment (cglewis) [inline check_files/requirements.txt]: unnecessary?

File: check_files/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+RUN apt-get update; apt-get install -y \
+    python3 \
+    libyaml-dev \
+    python3-dev \
+    python-dev \
+    python-pip \
+    clamav \
+    clamav-freshclam
+
+RUN pip install watchdog
```
Comment (cglewis) [inline check_files/Dockerfile]: unnecessary?

File: check_files/requirements.txt (hunk 2 above)
Comment (d-grossman) [reply check_files/requirements.txt]: it is now.. I'll get rid of it.

File: check_files/Dockerfile (hunk 3 above)
Comment (d-grossman) [reply check_files/Dockerfile]: it is now.. Ill get rid of it.

File: check_files/vent.template [hunk 4]
```diff
@@ -0,0 +1,9 @@
+[info]
+name = check_files
+groups = cpu
+
+[settings]
+ext_types = *.exe
```
Comment (cglewis) [inline check_files/vent.template]: `*.exe` -> `exe`

Comment (d-grossman) [reply check_files/vent.template]: thanks!

**Pull request merged**